packages/astro/types/actions.d.ts

```diff
@@ -1,3 +1,7 @@
 declare module 'astro:actions' {
+	export {
+		/** @deprecated Import `z` from `astro:schema` instead. */
+		z,
+	} from 'astro/zod';
```

Comment:
I don't understand this. The changeset says it's removed, but are we adding it as deprecated here?

Reply:
Ah, the changeset says it _will be removed in the next minor,_ hence why it's deprecated now. I know we're a few days from release, but it would be nice to get this in a little ahead with a clear deprecation note for early birds

Reply:
Update: it sounds like this is a minor release anyways, so let's just remove the `z` export in this PR.